ceph-build/build/build_deb

```diff
@@ -58,11 +58,12 @@ gen_debian_version() {
 bpvers=`gen_debian_version $debian_version $DIST`
 
 # look for a specific package to tell if we can avoid the build
-chacra_endpoint="ceph/${chacra_ref}/${distro}/${DIST}/${ARCH}/librados2_${vers}-${bpvers}_${ARCH}.deb"
+chacra_endpoint="ceph/${chacra_ref}/${distro}/${DIST}/${ARCH}"
+chacra_check_url="${chacra_endpoint}/librados2_${vers}-${bpvers}_${ARCH}.deb"
```

Comment:
rpm? this is for debs only. We don't need an endpoint pointing to a binary, we need one pointing to `ARCH`